README.md

```diff
@@ -364,6 +364,18 @@ with double quotes and sometimes it is necessary.
 
 - For `initscripts`,  `mac` is the currently configured MAC address of the device (`HWADDR`).
 
+### `cloned_mac`
+
+The `cloned_mac` address is optional and allow to specify the strategy to get the default
+mac or to set your own mac. The value of the `cloned_mac` address needs to be specified in
+hexadecimal notation like `mac` property. Besides explicitly specifying the value as a MAC
+address with hexadecimal notation, the following special values are also supported:
+
+- `permanent`: use the permanent MAC address of the device
+- `preserve`: don’t change the MAC address of the device upon activation
```

Comment:
The quote here needs to be changed according to the CI checks.


```
Running sanity test "no-smart-quotes"
ERROR: Found 2 no-smart-quotes issue(s) which need to be resolved:
ERROR: docs/README_network.md:375:18: use ASCII quotes `'` and `"` instead of Unicode quotes
ERROR: roles/network/README.md:375:18: use ASCII quotes `'` and `"` instead of Unicode quotes
See documentation for help: https://docs.ansible.com/ansible-core/2.12/dev_guide/testing/sanity/no-smart-quotes.html
```